### PR TITLE
Added Publishing the Mod tutorial section and example files

### PR DESCRIPTION
**Pull request by TomBock**
Added a Publishing the Mod tutorial section 
Added a TOC
Added example files for the Mod upload

### CODE REVIEW

File: Getting Started.md [hunk 1]
```diff
@@ -839,6 +885,116 @@ You should now be able to build and run this perk to see it at work.
...
+
+- Your mod .dll file (e.g. `MyFirstMod.dll`)
+- A Manifest file (`manifest.json`)
+- An icon of your choice, which will be displayed with the Mod name for others to see (`icon.png`)
+- A Readme file (`README.md`)
+
+Optionally you can add the following files:
+- A Changelog file (`CHANGELOG.md`)
+
+Note, that the exact filenames are required. If you encounter issue, the Thunderstore page will let you know what's missing.
```
Comment (JoyfulShush) [inline Getting Started.md]: Missing `an` after `encounter`.

File: Getting Started.md [hunk 2]
```diff
@@ -839,6 +885,116 @@ You should now be able to build and run this perk to see it at work.
 
 This should be a proper introduction into effects and their powers. Feel free to discuss with us on the our modding Discord if you need any assistance.
 
+# Publishing The Mod
+
+Once you are happy with your Mod, it is time to publish it to the world. This example will show you how to publish your Mod to the [Thunderstore ATS Mod Page](https://thunderstore.io/c/against-the-storm/).
```
Comment (JoyfulShush) [inline Getting Started.md]: For consistency, use `mod` in lowercase. Do so in all instances.

File: Getting Started.md (hunk 1 above)
Comment (TomBock) [reply Getting Started.md]: Added it.

File: Getting Started.md (hunk 2 above)
Comment (TomBock) [reply Getting Started.md]: Adjusted.

**Pull request merged**